### PR TITLE
Update release notes for 2024.04.1+748.pro2

### PR DESCRIPTION
**Pull request by jeffvroom**

### CODE REVIEW

**Review by petetronic — APPROVED** [review]
Those refer to the two issues included in the patch. Approving.

File: docs/news/index.qmd [hunk 1]
```diff
@@ -2,6 +2,36 @@
...
 
+## RStudio 2024.04.1
+
+**"Chocolate Cosmos"**
+
+>Date: 2024-05-11
+
+
+### New
+
```
Comment (MariaSemple) [inline docs/news/index.qmd:13]: Could we add a bullet here that we added support for R 4.4.0?
```suggestion

- Added support for R 4.4.0

```

File: docs/news/index.qmd [hunk 2]
```diff
@@ -2,6 +2,36 @@
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2024.04.1
+
+**"Chocolate Cosmos"**
+
+>Date: 2024-05-11
```
Comment (petetronic) [inline docs/news/index.qmd:9]: One question, is this date intended to match the binaries date or the date of the release?

Comment (jeffvroom) [reply docs/news/index.qmd:9]: Looks like it should be the release date so I updated it to be today.

**Pull request merged**